.github/workflows/release-workflow.yaml

```diff
@@ -18,11 +12,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
         with:
-         submodules: recursive
+          submodules: recursive
```

Comment:
Same question as above. Do we still need this?

Reply:
Removed